packages/frontend/src/hooks/useEcharts.ts

```diff
@@ -135,12 +136,18 @@ const valueFormatter =
         return getFormatterValue(rawValue, yField, getDimensions(explore));
     };
 
+const formatter = (format: string, field: string, data: any) => {
```

Comment:
Why is the wrapper needed? When is the value under `data.value[field]` vs. `data` ?

Reply:
Depending on the axis, apparently, axis Y returns an object, but axis X returns just the value

Reply:
I think I prefer to inline it or have separate functions, otherwise the function's purpose is confusing